### PR TITLE
Remove JCenter which is about to be sunset

### PR DESCRIPTION
**Pull request by iNikem**
And we had several JCenter timeouts today...

### CODE REVIEW

File: instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle [hunk 1]
```diff
@@ -76,5 +76,5 @@ dependencies {
 
   mojarra2TestImplementation group: 'org.glassfish', name: 'jakarta.faces', version: '2.3.12'
 
-  latestDepTestImplementation group: 'org.glassfish', name: 'jakarta.faces', version: '2+'
+  latestDepTestImplementation group: 'org.glassfish', name: 'jakarta.faces', version: '2.3.14'
```
Comment (mateuszrzeszutek) [inline instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle:79]: Should muzzle also be changed?

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by jkwatson — APPROVED** [review]

File: instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle (hunk 1 above)
Comment (iNikem) [reply instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle:79]: Yeah.. doing it now...

But MavenCentral metadata is broken :( we are doomed :(

Comment (mateuszrzeszutek) [reply instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle:79]: 😱

**Review by trask — APPROVED** [review]

File: instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle (hunk 1 above)
Comment (trask) [reply instrumentation/jsf/mojarra-1.2/javaagent/mojarra-1.2-javaagent.gradle:79]: 😭

File: instrumentation/vertx-web-3.0/vertx-web-3.0.gradle [hunk 2]
```diff
@@ -4,7 +4,7 @@ muzzle {
   pass {
     group = 'io.vertx'
     module = 'vertx-web'
-    versions = "[3.0.0,)"
+    versions = "[3.0.0,4.0.0)"
```
Comment (mateuszrzeszutek) [inline instrumentation/vertx-web-3.0/vertx-web-3.0.gradle:7]: Does `assertInverse = true` work here?

**Review by mateuszrzeszutek — APPROVED** [review]

File: instrumentation/vertx-web-3.0/vertx-web-3.0.gradle (hunk 2 above)
Comment (iNikem) [reply instrumentation/vertx-web-3.0/vertx-web-3.0.gradle:7]: It does. Added

**Review by mateuszrzeszutek — APPROVED** [review]

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -145,7 +145,7 @@ jobs:
         with:
           java-version: 11
       - name: Run  muzzle
-        run: ./gradlew ${{ matrix.module }}:muzzle --no-daemon
+        run: ./gradlew ${{ matrix.module }}:muzzle --no-daemon --no-build-cache -i
```
Comment (trask) [inline .github/workflows/pr.yaml:148]: is this change intended for merging?

**Review by trask — APPROVED** [review]
thank u 🦸‍♂️

File: .github/workflows/pr.yaml (hunk 3 above)
Comment (iNikem) [reply .github/workflows/pr.yaml]: removed, thanks :)
